### ISSUE TITLE
A11y_Azure Tools Storage Explorer_Signing in to Azure Storage_Toggle Explorer Pane - Tree View_ScreenReader:Clicking on the items that are empty inside , NVDA doesn't announce that the items are empty.

### ISSUE THREAD

**Issue by ManishaC1** [microsoft/AzureStorageExplorer#5712]
**GitHub Tags:**
#A11yMAS; #A11yTCS; #Win11; #DesktopApp; #A11ySev2; #BM_Azure Tools Storage Explorer_May2022; #Azure Tools Storage Explorer; #WCAG4.2.2; #ScreenReader; #NVDA; #FTP; #Benchmark; 

## Actual results:
When we press enter on the items that looks like it has some contents but actually are empty inside, for those item NVDA doesn't announce that the items are empty. 
For example, under the subscription named "AML V1 Personal 2" there is storage account called "coreuibrazileso1226133676", on expanding which we can see "queue" which looks like it has some contents in it which is because of the collapsed symbol. If we press enter on it, NVDA doesn't say that the item is empty.

## Expected results:
NVDA should announce the items are empty after clicking on "enter" for those items that are empty inside.

## Environmental Details:
Application: Microsoft Azure Storage Explore
version: 1.24.0

## OS Details
Microsoft Windows 11 Enterprise
version 21H2 OS Build 22000.613

## Repro Steps:

1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Open Explorer" button.
4. Traverse through the tree view opened on left of screen for whatever tasks or subscription selected.
5. Turn on the NVDA.
6. Verify that all the elements are accessible and MAS Compliant.

## User Impact:
Screen Reader might not get the information that the items are empty if screen reader doesn't announce it.

"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier."

## Attachments

https://user-images.githubusercontent.com/105198219/168733545-51205123-03c0-4654-859a-c5533b61f449.mp4

**Comment by MRayermannMSFT** [comment]
Hi @ManishaC1 

Please verify that this bug is fixed using the Storage Explorer build at: https://storageexplorerpublish.blob.core.windows.net/privatebuilds/20220624.3-accessibility-testing/StorageExplorer-ia32.exe

**Comment by ManishaC1** [comment]
Issue is not reproducing in the given environment. Hence closing the bug. Please see the below attachment.
![5712_Issue not repro](https://user-images.githubusercontent.com/105198219/175861735-cd16deb2-46da-4727-a782-76e64d23aa63.png)

**Comment by ManishaC1** [comment]
Github tags:#Closed

**Comment by Vishnuvarthancr7** [comment]
GitHubTags:#Closed;
